encrypted-media-respec.html

```diff
@@ -2544,20 +2544,19 @@ <h2>
               <p>
                 The robustness level associated with the content type. The empty string indicates
                 that any ability to decrypt and decode the content type is acceptable.
+                Implementations MUST configure the [=CDM=] to support at least the robustness
+                levels specified in the configuration of the {{MediaKeySystemAccess}} object used
+                to create the {{MediaKeys}} object.
               </p>
               <div class="note">
                 <p>
-                  Implementations MUST configure the [=CDM=] to support at least the robustness
-                  levels specified in the configuration of the {{MediaKeySystemAccess}} object used
-                  to create the {{MediaKeys}} object. Exact configuration of the [=CDM=] is
-                  implementation-specific, and implementations MAY configure the [=CDM=] to use the
-                  highest robustness level in the configuration even if a higher robustness level
-                  is available. If only the empty string is specified, implementations MAY be
-                  configured to use the lowest robustness level the implementation supports.
+                  Exact configuration of the [=CDM=] is implementation-specific.
+                  If only the empty string is specified, implementations have the option to
+                  use the lowest robustness level the implementation supports.
                 </p>
                 <p>
-                  Applications SHOULD specify the robustness level(s) they require to avoid
-                  unexpected client incompatibilities.
+                  Applications are recommended to specify the robustness level(s) they require,
+                  if any, to avoid unexpected client incompatibilities.
```

Comment:
```suggestion
                  To avoid unexpected interoperability issues, developers are encouraged to provide the robustness level(s) 
                  (if any).
```